app/main/helpers/login_helpers.py

```diff
@@ -3,6 +3,11 @@
 from flask_login import current_user
 from flask import request, redirect, url_for
 
+from datetime import datetime
+
+# Go Live Dates
+DOS6_GO_LIVE_DATE = '2022-01-14'
```

Comment:
As this is going to be a ~1 off, I think we're fine to hard-code it.

However, is this the right place for it? Should it live in config?

Reply:
That does seem more sensible the more I think about it, I shall move it in